docs/src/content/docs/components/Basic/checkbox.mdx

```diff
@@ -0,0 +1,229 @@
+---
+title: Checkbox
+---
+
+The `Checkbox` component is used to create a checkbox element in the UI.
+
+## Usage
+
+The `Checkbox` component allows the user to select or deselect a value.
+
+```tsx
+import Checkbox from '@components/Basic/Checkbox/Checkbox';
+
+const App = () => {
+    return (
+        <>
+            <Checkbox value="v-sync">V-Sync</Checkbox>
+            <Checkbox value="fullscreen" checked>Fullscreen Mode</Checkbox>
+            <Checkbox value="motion blur" checked>Motion Blur</Checkbox>
+            <Checkbox value="anti-aliasing">Anti-Aliasing</Checkbox>
+        </>
+    );
+};
+
+export default App;
+```
+
+## API
+
+### Props
+
+| Prop Name  | Type                                   | Default     | Description                                                                                                                                                                                  |
+| ---------- | -------------------------------------- | ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `style`    | `JSX.CSSProperties`                    | `{}`        | Inline styles to apply directly to the component's root element.                                                                                                                             |
+| `class`    | `string`                               | `""`        | Additional CSS classes to apply to the component                                                                                                                                             |
+| `ref`      | `CheckboxRef  \| undefined`            | `undefined` | A reference to the component that gives you access to its methods and the underlying HTML element. Useful if you need to toggle the checkbox programmatically.                                                             |
+| `disabled` | `boolean`                              | `false`     | Specify if the checkbox is disabled                                                                                                                                                            |
+| `value`    | `any`                                  | `''`        | The value associated with the checkbox component. |
+| `checked`  | `boolean`                              | `false`     | Specify if the checkbox is checked initially. |
+| `onChange` | `(checked: boolean) => void`           | `undefined` | A function that is called every time the checkbox is toggled. It can be used to retrieve whether the checkbox is checked. |
+
+### Methods
+
+|Method |Parameters |Return Value |Description |
+|---|---|---|---|
+| `toggle` | `(e?: MouseEvent) => void`  | `void` | Toggles the checkbox |
```

Comment:
And what if I want to just uncheck it? In the example below you mentioned that this method is used to toggle the checkbox. And why we have here `(e?: MouseEvent) => void`? Maybe instead we need to expose setChecked method to the ref?